src/editors/content/learning/conjugation/ConjugationEditor.tsx

```diff
@@ -211,8 +212,10 @@ class ConjugationEditor
     const columnsToAdd = this.props.model.rows.last().cells.size;
 
     const kvPairs = [];
-    for (let i = 0; i < columnsToAdd; i += 1) {
-      const cell = new contentTypes.CellHeader();
+    const ch = new contentTypes.CellHeader();
```

Comment:
We were adding all cell headers when a new row was created. I changed it to be a single header and then conjugates